docs/tour/hello_world.mdx

```diff
@@ -24,215 +23,190 @@ If you have not installed `wash`, follow the [installation guide](../installatio
 
-### Running a capability provider
+:::info
 
-For this actor to receive HTTP requests, we need to start the HTTP Server capability provider.
-Actors are signed WebAssembly modules, and as such they have embedded claims declaring their ability
-to communicate with capability providers like the HTTP Server. Actors cannot communicate with any
-capability provider for which they have not been signed.
+Capabilities are _deny by default_, and if you take a look in `wasmcloud.toml` you'll see the only capability is `wasmcloud:httpserver`. This means that this actor can never access files on disk, make network requests of its own, or even access the system time, it's blocked by the wasmCloud runtime.
 
-Let's use `wash` to inspect the set of capabilities this actor has:
+:::
 
-```shell
-wash claims inspect wasmcloud.azurecr.io/echo:0.3.8
+When you ran `wash app deploy wadm.yaml`, `wadm` takes the application manifest and schedules the components on hosts according to the scale you specify. This example has a minimal manifest, let's take a look at it:
+
+```yaml
+# Metadata
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: hello
+  annotations:
+    version: v0.0.1
+    description: "wasmCloud Hello World Example"
+spec:
+  components:
+    # Your actor component, started from the wasmCloud example OCI artifact
+    - name: hello
+      type: actor
+      properties:
+        image: wasmcloud.azurecr.io/hello:0.1.7
+      traits:
+        # One replica of this actor will run
+        - type: spreadscaler
+          properties:
+            replicas: 1
+        # The link configuration tells the httpserver to listen on 8080
+        - type: linkdef
+          properties:
+            target: httpserver
+            values:
+              address: 0.0.0.0:8080
+
+    # The httpserver capability provider, started from the official wasmCloud OCI artifact
+    - name: httpserver
+      type: capability
+      properties:
+        image: wasmcloud.azurecr.io/httpserver:0.17.0
+        contract: wasmcloud:httpserver
 ```
 
-```shell
-                               Echo - Module
-  Account       ACOJJN6WUP4ODD75XEBKKTCCUJJCY5ZKQ56XVKYK4BEJWGVAOOQHZMCW
-  Module        MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5
-  Expires                                                          never
-  Can Be Used                                                immediately
-  Version                                                      0.3.8 (4)
-  Call Alias                                                   (Not set)
-                               Capabilities
-  HTTP Server
-  Logging
-                                   Tags
-  None
+## Scaling up 📈
+
+WebAssembly can be easily scaled due to its small size (this actor clocks in at `201kb`) and it's easy to scale your applications with wasmCloud. Let's scale up our hello world application to 50 replicas by editing `wadm.yaml`:
```

Comment:
It's not clear whether this is on-disk size or in-memory size. For scaling we should be highlighting memory footprint

Reply:
Going to step away from specific numbers as I would rather quote confident benchmarks than try and measure something locally, despite seeing really impressive numbers (I started 100 instances of this actor and my passive memory usage only increased by 8MB, eventually falling even lower thanks to garbage collection I think)

Reply:
Let me know what you think of the changes